juju-charms/charm-trilio-data-mover/src/reactive/trilio_data_mover.py

```diff
@@ -356,16 +356,13 @@ def create_service_file():
     tv_config.add_section('Install')
     tv_config.set('Unit', 'Description', 'TrilioVault DataMover')
     tv_config.set('Unit', 'After', 'openstack-nova-compute.service')
-    tv_config.set('Unit', 'StartLimitBurst', 5)
-    tv_config.set('Unit', 'StartLimitIntervalSec', 60)
```

Comment:
Removed these fields as its names and corresponding section depend upon the systemd version. We will take the default values.